packages/files-ui/src/Components/Modules/FileBrowsers/hooks/useLookupUser.tsx

```diff
@@ -29,9 +39,10 @@ export const useLookupSharedFolderUser = () => {
 
       if (!result) return []
 
-      const currentUsers = Array.isArray(sharedFolderUsers)
-        ? sharedFolderUsers.map(su => su.value)
-        : []
+      if (!result) return []
+      const usersList = permission === "read" ? sharedFolderReaders : sharedFolderWriters
+      const currentUsers = Array.isArray(usersList) ? usersList.map(su => su.value) : []
```

Comment:
No need to check if it's an array, the type indicated that it must be.
```suggestion
      const currentUsers = usersList.map((su) => su.value)
```

Reply:
you forgot about this one, committing it now.